commands/templates/svc/manifest-public.yml

```diff
@@ -30,25 +27,28 @@ sidecars:
     port: 443
     image: public.ecr.aws/uktrade/nginx-reverse-proxy:latest
     variables:
-      SERVER: localhost:{% if ipfilter %}8000{% else %}8080{% endif %}
+      SERVER: localhost:8000
 
-{% if ipfilter %}
   ipfilter:
     port: 8000
     image: public.ecr.aws/uktrade/ip-filter:latest
     variables:
       PORT: 8000
-      EMAIL_NAME: 'The Department for International Trade WebOps team'
-      EMAIL: test@test.test
-      LOG_LEVEL: DEBUG
+      EMAIL_NAME: DIT
+      EMAIL: sre@digital.trade.gov.uk
       ORIGIN_HOSTNAME: localhost:8080
-      ORIGIN_PROTO: http
-      CONFIG_FILE: 's3://ipfilter-config/ROUTES.yaml'
-{% endif %}
+      APPCONFIG_PROFILES: ipfilter:default:default
+      IPFILTER_ENABLED: {{ ipfilter }}
+
+  appconfig:
+    port: 2772
+    image: public.ecr.aws/aws-appconfig/aws-appconfig-agent:2.x
+    essential: true
+    variables:
+      ROLE_ARN: arn:aws:iam::763451185160:role/AppConfigIpFilterRole
```

Comment:
What are the security implications of leaving the account ID?

Reply:
The account number alone is not directly exploitable, and the ARN should not be either - as long as it's configured correctly.  It's still not ideal to expose this though.